skipper.go

```diff
@@ -861,6 +865,7 @@ func Run(o Options) error {
 			loadbalancer.HealthcheckPostProcessor{LB: lbInstance},
 			loadbalancer.NewAlgorithmProvider(),
 		},
+		PreProcessors:   []routing.PreProcessor{o.DefaultFilters},
```

Comment:
we should not add the o.DefaultFilters when it is nil

Reply:
fixed